modules/mbedtls/Kconfig.tls-generic

```diff
@@ -559,6 +559,16 @@ config MBEDTLS_PSA_P256M_DRIVER_ENABLED
 	  Enable support for the optimized sofware implementation of the secp256r1
 	  curve through the standard PSA API.
 
+config MBEDTLS_PSA_KEY_SLOT_COUNT
+	int "Number of key slots in PSA core"
+	default 32
+	depends on MBEDTLS_PSA_CRYPTO_C
```

Comment:
Can we instead group all the Kconfig options that have this condition inside an `if`?